docs/source/guides/saving_and_loading_dataframes.ipynb

```diff
@@ -215,25 +215,37 @@
     "from woodwork import read_file\n",
     "\n",
     "woodwork_df = read_file(\n",
-    "    filepath='retail.csv',\n",
+    "    filepath='retail/data.csv',\n",
     "    content_type='csv',\n",
     "    **typing_information,\n",
     ")\n",
     "\n",
     "woodwork_df = read_file(\n",
-    "    filepath='retail.parquet',\n",
+    "    filepath='retail/data.parquet',\n",
     "    content_type='parquet',\n",
     "    **typing_information,\n",
     ")\n",
     "\n",
     "woodwork_df = read_file(\n",
-    "    filepath='retail.feather',\n",
+    "    filepath='retail/data.feather',\n",
     "    content_type='feather',\n",
     "    **typing_information,\n",
     ")\n",
     "\n",
     "woodwork_df.ww"
    ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Let's hide this cell
- https://github.com/alteryx/featuretools/issues/1560#issuecomment-887799238

Reply:
✅ 8b68909